src/reducers/settings.js

```diff
@@ -0,0 +1,19 @@
+import { handleActions } from 'redux-actions'
+
+import { updateSettings } from 'actions/settings'
+
+const reducer = handleActions({
+  [updateSettings]: (state, action) => {
+    console.log('REDUX ', state)
+    const data = action.payload
+    let result = {}
+    data.map((item) => {
+      if (item.address && item.name) {
+        result[item.address] = item.name
+      }
+    })
+    return result
```

Comment:
I would like to already nest the mappings here within the settings. I think we're going to use settings for more than just the whitelist mapping later on, maybe it would be better if the reducer wrote into `settings.mapping`? On Line 17 you can initialize the reducer with an empty object as the mapping (`, { mapping: {} }`)